packages/flame_tiled/test/tiled_test.dart

```diff
@@ -40,6 +40,13 @@ void main() {
       expect(tiled.tileMap.renderableLayers.length, equals(3));
     });
 
+    test('correct loads the file, with different prefix', () async {
+      tiled = await TiledComponent.load('map_custom_path.tmx', Vector2.all(16),
+          prefix: 'assets/tiles/tiles_custom_path/');
```

Comment:
```suggestion
          prefix: 'assets/tiles/tiles_custom_path/',);
```
Just add a trailing comma here and re-format

Reply:
done